openedx/core/djangoapps/call_stack_manager/tests/tests.py

```diff
@@ -0,0 +1,162 @@
+"""
+Test cases for Call Stack Manager
+"""
+from .models import ModelMixin, ModelNothing, ModelMixinCSM, ModelAnotherCSM
+from django.test import TestCase
+from testfixtures import LogCapture
+from openedx.core.djangoapps.call_stack_manager import donottrack
+
+
+class TestingCallStackManager(TestCase):
+    """
+    Tests for call_stack_manager package
+    """
+    def setUp(self):
+        super(TestingCallStackManager, self).setUp()
+
+    def tearDown(self):
+        """ Deleting all databases after tests
+        """
+        super(TestingCallStackManager, self).tearDown()
+        ModelMixin.objects.all().delete()
+        ModelMixinCSM.objects.all().delete()
+        ModelNothing.objects.all().delete()
+        ModelAnotherCSM.objects.all().delete()
```

Comment:
You shouldn't need to do this. The django `TestCase` cleans up after any models used in the tests.